doc-content/enterprise-only/installation/clustering-dev-con.adoc

```diff
@@ -20,10 +20,8 @@ ifdef::PAM[]
  to test those rules and processes.
 endif::[]
 
-To create a {PRODUCT} clustered development environment, you download and extract {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION}. Then, you configure {EAP} {EAP_VERSION} with Elasticsearch and AMQ Broker, a Java messaging server (JMS) broker. Finally, you start the cluster and install {CENTRAL} on each cluster node.
+To create a {PRODUCT} clustered development environment, you download and extract {EAP} {EAP_VERSION} and {PRODUCT} {PRODUCT_VERSION}. Then, you configure {EAP} {EAP_VERSION} with {DATAGRID} {DATAGRID_VERSION_LONG} and AMQ Broker, a Java messaging server (JMS) broker. Finally, you start the cluster and install {CENTRAL} on each cluster node.
```

Comment:
This entire introductory section can be written better. It starts with calling the user (System Engineers in this case, as specified on top of the document) a Developer X (with a capital D). There is no mention of Developer Y, Z or any other, so the whole point of calling the user Developer X seems baseless. 
The second paragraph has an unnecessary full stop in between the sentence.
The third paragraph seems like a misfit here since it doesn't flow well from the first two. In fact there is a complete disconnect of all the 5 paragraphs with each other.

Reply:
Rewritten. Retained developer and user as the key personas, and "you" refers to the system engineer. WDYT?

You can configure {PRODUCT} as a clustered development environment to benefit from high availability. With a clustered environment, if a developer is working on Node1 and that node fails, that developer's work is preserved and visible on any other node of the cluster.